source/stylesheets/_refills.scss

```diff
@@ -0,0 +1,16 @@
+.refill {
+  border-top: 2px solid $base-border-color;
+  margin-bottom: $base-spacing * 6;
+  padding-top: $base-spacing;
+}
+
+.refill-wrapper {
+  align-items: center;
+  background-color: tint($base-border-color, 90%);
+  border: $base-border;
+  display: flex;
+  justify-content: space-around;
+  margin-top: $base-spacing;
+  min-height: $base-spacing * 12;
```

Comment:
I could’ve been more clear. I always trying to indicate that a system of variables, based on `$base-spacing` should be available and used within components. Any spacing values that are used that are not part of the system, would be a smell that the system is broken.

This is something Bitters should do. I have a PR up that attempts to start solving that: https://github.com/thoughtbot/bitters/pull/266